src/assets/messages.json

```diff
@@ -90,8 +88,6 @@
     "CollaborativeFilteringRatingBlock": "Collaborative Filtering Rating",
     "CollaborativeFilteringRatioBranchBlock": "Branch via Collaborative Filtering Ratio",
     "ConnectToOperatorBlock": "Connect to Operator",
-    "ConsoleIO.Print": "Print",
-    "ConsoleIO.Read": "Read",
```

Comment:
I wonder if we should also remove the strings from [the localization sheet](https://docs.google.com/spreadsheets/d/136-y0DYRWFy6Max-Xz2AsqJ95TQvwrKjal6Wn-z0lsI/edit#gid=561815572).

Reply:
Good point.
@safydy-r what's your opinion?

Reply:
@bulatgab , if we're sure those localization are not used, I'd encourage us to remove them in the localization sheet. Maintenance like this would keep the sheet small ;) 

- [ ] let's remove them

Reply:
I've removed those rows
(I've left a copy of them here just in case: https://docs.google.com/spreadsheets/d/1_sul1YRaMpaADcBbs2P6hk0f3LMXVVEqBJHF3fB0KWk/edit?usp=sharing)

Reply:
I've run the script in voto5 to update the localizations but it didn't remove those lines.
I suspect that the script only adds and doesn't remove. If you're okay with it, I'd suggest we ignore this.
Maybe the script gets improved in the future.